partiql-tests-data/eval/primitives/functions/postgresql-multibyte.ion

```diff
@@ -0,0 +1,1489 @@
+            attr: "コンピュータディスプレイ",
+            result: 12
+          },
+          {
+            attr: "コンピュータグラフィックス",
+            result: 13
+          },
+          {
+            attr: "コンピュータプログラマー",
+            result: 12
+          },
+        ]
+      }
+    },
+  ],
+  octet_length::[
+    {
+      name:"OCTET_LENGTH gb18030",
+      statement:"SELECT attr, OCTET_LENGTH(attr) AS result FROM tbl",
+      assert:{
+        result:EvaluationSuccess,
+        evalMode:[
+          EvalModeCoerce,
+          EvalModeError
+        ],
+        output:$bag::[
+          {
+            attr: "コンピュータディスプレイ",
+            result: 36
+          },
+          {
+            attr: "コンピュータグラフィックス",
+            result: 39
+          },
+          {
+            attr: "コンピュータプログラマー",
+            result: 36
+          },
+        ]
+      }
+    },
+  ],
+  position::[
+    {
+      name:"POSITION gb18030",
+      statement:"SELECT attr, POSITION('デ' IN attr) AS result FROM tbl",
+      assert:{
+        result:EvaluationSuccess,
+        evalMode:[
+          EvalModeCoerce,
+          EvalModeError
+        ],
+        output:$bag::[
+          {
+            attr: "コンピュータディスプレイ",
+            result: 7
+          },
+          {
+            attr: "コンピュータグラフィックス",
+            result: 0
+          },
+          {
+            attr: "コンピュータプログラマー",
+            result: 0
+          },
+        ]
+      }
+    },
+  ],
+  substring::[
+    {
+      name:"SUBSTRING gb18030",
+      statement:"SELECT attr, SUBSTRING(attr FROM 10 FOR 4) AS result FROM tbl",
+      assert:{
+        result:EvaluationSuccess,
+        evalMode:[
+          EvalModeCoerce,
+          EvalModeError
+        ],
+        output:$bag::[
+          {
+            attr: "コンピュータディスプレイ",
+            result: "プレイ"
+          },
+          {
+            attr: "コンピュータグラフィックス",
+            result: "ィックス"
+          },
+          {
+            attr: "コンピュータプログラマー",
+            result: "ラマー"
+          },
+        ]
+      }
+    },
+  ],
+]
+
+mule_internal::[
+  envs::{
+    tbl1:[
+      {
+        attr: "’¥³’¥ó’¥Ô’¥å’¡¼’¥¿’¥Ç’¥£’¥¹’¥×’¥ì’¥¤"
+      },
+      {
+        attr: "’¥³’¥ó’¥Ô’¥å’¡¼’¥¿’¥°’¥é’¥Õ’¥£’¥Ã’¥¯’¥¹"
+      },
+      {
+        attr: "’¥³’¥ó’¥Ô’¥å’¡¼’¥¿’¥×’¥í’¥°’¥é’¥Þ’¡¼"
+      },
+    ],
+    tbl2:[
+      {
+        attr: "‘µç‘ÄÔ‘ÏÔ‘Ê¾‘ÆÁ"
+      },
+      {
+        attr: "‘µç‘ÄÔ‘Í¼‘ÐÎ"
+      },
+      {
+        attr: "‘µç‘ÄÔ‘³Ì‘Ðò‘Ô±"
+      },
+    ],
+    tbl3:[
+      {
+        attr: "“ÄÄ“Ç»“ÅÍ“µð“½º“ÇÃ“·¹“ÀÌ"
+      },
+      {
+        attr: "“ÄÄ“Ç»“ÅÍ“±×“·¡“ÇÈ“½º"
+      },
+      {
+        attr: "“ÄÄ“Ç»“ÅÍ“ÇÁ“·Î“±×“·¡“¸Ó"
+      },
+    ],
+    tbl4:[
+      {
+        attr: "ENGLISH"
+      },
+      {
+        attr: "FRANÇAIS"
+      },
+      {
+        attr: "ESPAÑOL"
+      },
+      {
+        attr: "ÍSLENSKA"
+      },
+      {
+        attr: "ENGLISH FRANÇAIS ESPAÑOL ÍSLENSKA"
+      },
+    ]
+  },
+  character_length::[
```

Comment:
These tests failed because they were specified with a different character set in postgres (see [here](https://www.postgresql.org/docs/current/multibyte.html#CHARSET-TABLE) for more on character sets in postgres). Moved all of the postgres multibyte tests to [partiql-tests-data-extended](https://github.com/partiql/partiql-tests/pull/44/files#diff-cd4eebbb02dc623e15445f66a7e7526a646702fbf9cde2fd94070a3cc0d4bfc9) directory, so they could be added when/if PartiQL supports other character sets.